valhalla/jawn/src/lib/handlers/LoggingHandler.ts

```diff
@@ -418,4 +458,63 @@ export class LoggingHandler extends AbstractLogHandler {
 
     return requestInsert;
   }
+
+  private pullRequestBodyMessage(requestBody: any): string {
+    try {
+      const messagesArray = requestBody.messages;
+
+      if (!Array.isArray(messagesArray)) {
+        return "";
+      }
+
+      const allMessages = messagesArray
+        .filter((message) => {
+          return message.role === "user";
+        })
+        .map((message) => {
+          if (typeof message === "object" && message !== null) {
+            return message["content"] || "";
+          }
```

Comment:
I think this is missing a few cases...  

<img width="534" alt="image" src="https://github.com/Helicone/helicone/assets/26822232/48d130b4-5621-48f3-8bfe-ed9eda943368">

Reply:
<img width="1125" alt="Screenshot 2024-05-30 at 2 47 14 PM" src="https://github.com/Helicone/helicone/assets/26822232/748b7f5b-8b4f-4336-afd4-c15985988426">


`content` can be an array

Reply:
https://platform.openai.com/docs/api-reference/chat/create